erpnext/accounts/doctype/account/account_tree.js

```diff
@@ -26,8 +26,6 @@ frappe.treeview_settings["Account"] = {
 			description: __("Name of new Account. Note: Please don't create accounts for Customers and Suppliers")},
 		{fieldtype:'Check', fieldname:'is_group', label:__('Is Group'),
 			description: __('Further accounts can be made under Groups, but entries can be made against non-Groups')},
-		{fieldtype:'Select', fieldname:'root_type', label:__('Root Type'),
```

Comment:
@saurabh6790 you should not delete this field, it should be shown conditionally, only for root account.